src/components/search/filters/Filters.js

```diff
@@ -96,6 +96,30 @@ function Filters({ loading, query, aggregations, applyFilter, ...rest }) {
                 emptyDisabled
                 {...filterProps}
             />
+
+            <TermsAggregationFilter
+                title="From"
+                field="from.keyword"
+                onLoadMore={handleLoadMore}
+                emptyDisabled
+                {...filterProps}
+            />
+
+            <TermsAggregationFilter
+                title="To"
+                field="to.keyword"
+                onLoadMore={handleLoadMore}
+                emptyDisabled
+                {...filterProps}
+            />
+
+            <TermsAggregationFilter
+                title="Path parts"
```

Comment:
Rename title to just "Path"


I know the archives appear twice in the bucket list (with and without / at end), i'll remove them from the backend some other time.